src/main/java/screens/ModifyTaskScreen.java

```diff
@@ -0,0 +1,97 @@
+package screens;
+
+import modify_task_use_case.ModifyTaskController;
+import modify_task_use_case.ModifyTaskFailed;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+
+/**
+ * Creates a screen with the form a user fills out to modify an existing task.
+ * Contains a text field autofilled with the original task name, a modify
+ * button, and a cancel button.
+ * @author Anna Myllyniemi
+ */
+public class ModifyTaskScreen extends FormScreen{
+    private final ModifyTaskController modifyTaskController;
+    private String oldTitle = "";
+    private int dayIndex = 0;
+
+    /**
+     * Creates a JPanel with a JTextField and two JButtons
+     *
+     * @param controller the controller to call when the user hits the modify button
+     */
+    public ModifyTaskScreen(ModifyTaskController controller) {
+        super("Modify Task");
+        modifyTaskController = controller;
+
+        JPanel panel = new JPanel(new GridBagLayout());
+        GridBagConstraints c = initializeConstraints();
+        addHeader(panel, c, "MODIFY THE TASK");
+        addTitleField(panel, c, 1, "Task Title:");
+        addFormButtons(panel, c, 3, "Modify");
+        this.add(panel);
+    }
+
+    /**
+     * Reset the form values to the default
+     */
+    @Override
+    protected void resetForm() {
```

Comment:
resetForm() needs to be protected because it is an abstract method in FormScreen and it is called in one of the window listener methods in FormScreen. If the method was private then it would not be accessible to the method that calls it from FormScreen.

Reply:
Makes sense. Thanks for the clarification.